modules/vaos/app/models/vaos/message_form.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require 'active_model'
+require 'common/models/form'
+
+module VAOS
+  class MessageForm < Common::Form
+    attribute :message_text, String # only this attribute gets set by user, everything else is overridden
+    attribute :url, String
+    attribute :sender_id, String
+    attribute :appointment_request_id, String
+    attribute :message_date_time, String
+    attribute :message_sent, Boolean
+    attribute :is_last_message, Boolean
+    attribute :_appointment_request_id, String
+
+    def initialize(user, request_id, json_hash = {})
+      super(json_hash)
+      @user = user
+      @appointment_request_id = request_id
+    end
+
+    # HACK: addresses the fact that downstream requires this as AppointmentRequestId (upper camelcase)
+    def _appointment_request_id
+      appointment_request_id
+    end
+
+    def url
+      ''
+    end
+
+    def sender_id
+      @user.icn
+    end
+
+    def message_date_time
+      ''
+    end
+
+    def message_sent
+      true
+    end
+
+    # rubocop:disable Naming/PredicateName
```

Comment:
You've got control of this variable name, so I'd recommend following the pattern and using `last_message`

Reply:
we do not have control of this name, need this to be sent to VAMF. In fact, if you look more closely we're overriding it and setting it manually such that it cannot be assigned by FE. This is a required field by VAMF to validate this form object.

Reply:
I don't love this but tbh it's not VSP code - a solution would have to change the name when the response is sent anyways.